srcpkgs/collectd/template

```diff
@@ -1,18 +1,22 @@
 # Template file for 'collectd'
 pkgname=collectd
 version=5.11.0
-revision=1
+revision=2
 build_style=gnu-configure
 configure_args="$(vopt_enable rrdtool rrdtool) $(vopt_enable perl perl)
  $(vopt_enable notify notify_desktop) --with-libiptc --enable-virt
  --sysconfdir=/etc --disable-static --with-fp-layout=nothing
- --with-python=/bin/python3 --disable-werror"
+ --with-python=/bin/python3 --disable-werror
+ $(vopt_with prometheus "libmicrohttpd protobuf-c")
+ $(vopt_enable prometheus write_prometheus)"
 conf_files="/etc/collectd.conf"
 hostmakedepends="automake pkg-config libtool libltdl-devel glib-devel"
 makedepends="libcurl-devel iptables-devel libressl-devel libsensors-devel
  libesmtp-devel zlib-devel yajl-devel python3-devel libxml2-devel net-snmp-devel
  libgcrypt-devel libmysqlclient-devel libltdl-devel libvirt-devel
- $(vopt_if notify libnotify-devel) $(vopt_if rrdtool rrdtool-devel)"
+ $(vopt_if notify libnotify-devel) $(vopt_if rrdtool rrdtool-devel)
+ $(vopt_if prometheus "protobuf-c-devel libmicrohttpd-devel")"
+depends="$(vopt_if prometheus protobuf-c libmicrohttpd)"
```

Comment:
Aren't these libraries detected automatically? And multiple items like these should be in single quotes: `'protobuf-c libmicrohhtpd')"`

Reply:
I thought the point was to ensure the dependent libraries are present if that option is selected?

Reply:
The idea is that when building with this option, the final binaries will be dynamically linked to these libraries, meaning that the build system automatically detects them.